src/Form/Formbot.js

```diff
@@ -146,16 +146,15 @@ export default class Formbot extends React.Component {
       let errorMsg;
 
       if (hasSchema && typeof validation.validate === 'function') {
-        validation
-          .validate(fieldValue, validationOpts)
-          .catch(e => {
-            errorMsg = e.message;
-          })
-          .finally(() => {
-            this.updateField(field, { validated: true }).then(() => {
-              this.setErrors({ [field]: errorMsg }, resolve);
-            });
+        try {
+          validation.validate(fieldValue, validationOpts);
+        } catch(e) {
```

Comment:
I don't think this is going to work actually `async` functions. This will be rejected with unhandled promise error because the `try/catch` block won't catch the promise rejection. We need to check if the result of the calling the function returns a promise or whether is just returns a value or throws.

Reply:
@erikshestopal Turns out this is a non-issue, the user was on an unsupported version of the browser.